spark3/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -491,6 +491,19 @@ public static int batchSize(Map<String, String> properties, CaseInsensitiveStrin
             TableProperties.PARQUET_BATCH_SIZE, TableProperties.PARQUET_BATCH_SIZE_DEFAULT));
   }
 
+  public static Boolean propertyAsBoolean(CaseInsensitiveStringMap options, String property, Boolean defaultValue) {
```

Comment:
Should this use `boolean` instead of `Boolean`?

Reply:
Yea I thought about it, but since other similar methods like `propertyAsLong`, `propertyAsInt` all return the boxed version, I did the same. I can change it for sure.

Reply:
I wonder why the others do. I'll have to take a look at it.